lib/bap_llvm/llvm_loader_scheme.hpp

```diff
@@ -0,0 +1,200 @@
+#ifndef LLVM_LOADER_SCHEME_HPP
```

Comment:
Do we really need to reimplement Ogre.ml in C++? The idea is that you just do

```c++
scheme << "(segment " << addr << " " << size << " " << name << ")";
```

You do a double work, the OCaml side will check that everything is fixed. The C++ will still suck. I think that this is an overengineering.